q1.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html >
+<head>
+  <title> Question 10 </title>
+  <script src ="./index.js"></script>
+</head>
+<body>
+  <h1>Disemvoweler</h1>
+
+  <label> Enter text here <br>
+      <input id = "string" type = "text"/> <br>
+  </label>
+  <input id = "yInclude" type="checkbox" name="vowel" value="" checked> Y is a vowel<br>
+  <button id = "submitButton">
```

Comment:
button tag needs a type
there is no space between id and = and = and ": id="submitButton"